content/en/logs/guide/restrict-access-to-Log-events-with-restriction-queries.md

```diff
@@ -0,0 +1,389 @@
+---
```

Comment:
Watch out the file name has `Log` uppercase in it, file names should be lowercase only

Reply:
Thanks fixed